go/v1/api/validators/basic_check.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validators
+
+import (
+	"strings"
+)
+
+const (
+	// MaxNoteIDLength stands for the max length allowed for a note_id field.
+	MaxNoteIDLength = 128
```

Comment:
How was this determined? Because this is in an OSS repository, it'd be good to provide some rationale for the choices of values here.

Reply:
the note_id will eventually be written to DB (instead of in-mem map).

DB often has this 128 limit for identifiers:
https://stackoverflow.com/questions/5808332/sql-server-maximum-character-length-of-object-names
https://oracle-base.com/articles/12c/long-identifiers-12cr2